tasks/run-buildpack-checksums-specs.yml

```diff
@@ -13,6 +13,10 @@ run:
     - |
       set -ex
       cd buildpack-checksums
+      if [ -n "${RUBYGEM_MIRROR}" ]; then
+        gem sources --clear-all --add ${RUBYGEM_MIRROR}
+      fi
+      gem install bundler --no-ri --no-rdoc
```

Comment:
There is only a few `gem install bundler` call in the codebase, so I'll do it.
```
scripts/binary-builder-unit.sh
9:  gem install bundler --no-ri --no-rdoc

scripts/binary-builder-integration.sh
9:  gem install bundler --no-ri --no-rdoc

Dockerfile
57:RUN gem install bosh_cli bosh_cli_plugin_micro rake bundler

tasks/run-buildpack-checksums-specs.yml
19:      gem install bundler --no-ri --no-rdoc

tasks/build-buildpack-checksums-site.yml
21:        gem install bundler
```
Do I also need to update the dockerfile ?